src/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/DesignerOptionService.cs

```diff
@@ -33,7 +33,7 @@ public DesignerOptionCollection Options
         /// <summary>
         ///     Creates a new DesignerOptionCollection with the given name, and adds it to 
         ///     the given parent.  The "value" parameter specifies an object whose public 
-        ///     properties will be used in the Propeties collection of the option collection.  
+        ///     properties will be used in the Properties collection of the option collection.  !!maybe small properties here?
         ///     The value parameter can be null if this options collection does not offer 
```

Comment:
Capital, it's referring to the `Properties` property on line 197.

`<see cref="Properties"/>` would be even better, as it would the link to that property, and likewise `<paramref name="value"/>` instead of `"value"` in the line above, though maybe such changes should be left to another PR, and just concentrate on spelling errors.